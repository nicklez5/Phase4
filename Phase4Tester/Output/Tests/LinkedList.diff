--- SelfTestCases/LinkedList.out	2011-04-28 14:55:21.000000000 -0700
+++ ./Output/Tests/LinkedList.out	2020-03-14 13:04:33.785787164 -0700
@@ -1,27 +1,4 @@
-25
-10000000
-39
-25
-10000000
-22
-39
-25
-1
-0
-10000000
-28
-22
-39
-25
-2220000
--555
--555
-28
-22
-25
-33300000
-22
-25
-44440000
-0
+Error in /home/jackson/Desktop/phase4/Phase4Tester/./Output/Tests/LinkedList.compiler.out line 338 column 6: "$s8": operand is of incorrect type
+Error in /home/jackson/Desktop/phase4/Phase4Tester/./Output/Tests/LinkedList.compiler.out line 461 column 6: "$s8": operand is of incorrect type
 
+Processing terminated due to errors.
