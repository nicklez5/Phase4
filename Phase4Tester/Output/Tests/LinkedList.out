Error in /home/jackson/Desktop/phase4/Phase4Tester/./Output/Tests/LinkedList.compiler.out line 338 column 6: "$s8": operand is of incorrect type
Error in /home/jackson/Desktop/phase4/Phase4Tester/./Output/Tests/LinkedList.compiler.out line 461 column 6: "$s8": operand is of incorrect type

Processing terminated due to errors.
